docs/auth/private-vs-public.mdx

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 1
+sidebar_label: Private vs Public
+description: Learn how to choose between private and public APIs in Hasura DDN.
+keywords:
+  - private
+  - public
+  - api
+  - authentication
+  - authorization
+---
+
+import Thumbnail from "@site/src/components/Thumbnail";
+
+# Private vs Public
+
+You can choose to make your Hasura DDN API public or private.
+
+## Private
+
+A `private` Hasura DDN API is only accessible to collaborators on your project.
+
+Queries to a private Hasura DDN API must include a special reserved header `x-hasura-ddn-token` with a valid JWT token
+which the Hasura console generates and regenerates every hour.
+
+If a `private` API is also set to JWT or Webhook mode, rather than `noAuth` mode, queries must **also** include the JWT
+or webhook authentication values to be successful in addition to the `x-hasura-ddn-token` header.
+
```

Comment:
After seeing a few internal discussions in Slack, it would be helpful to make abundantly clear that this is not meant for production projects. While it's true a private project can be hit with an `x-hasura-ddn-token` header, we need to take care not to advocate for this use case, as it wasn't designed as an auth method.

Reply:
Emphasised that the token is only available and regenerated in the console and an explicit note about private mode not being for production.